oasislmf/utils/data.py

```diff
@@ -429,13 +428,38 @@ def get_ids(df, usecols, group_by=[]):
     :param group_by: A subset of the column subset to use a subgroup key
     :param group_by: list
 
+    :param allow_duplicates: allow non-unique ids in return
+    :param allow_duplicates: Boolean
+
```

Comment:
Parameter `allow_duplicates` not an argument in `get_ids` function. Suggest removing it to avoid confusion:
```suggestion
```